src/rgw/rgw_sal_motr.cc

```diff
@@ -2497,8 +2497,7 @@ int MotrObject::get_bucket_dir_ent(const DoutPrefixProvider *dpp, rgw_bucket_dir
 
 out:
   if (rc == 0) {
-    sal::Attrs dummy;
-    decode(dummy, iter);
+    decode(attrs, iter);
```

Comment:
How this solving the problem, please add some justification

Reply:
Get Object API  was not able to get the attributes present in attrs, hence updated the final goto block wherein the attributes present in ent will be decoded and put in attrs. Hence GetObject API which calls MotrObject::get_bucket_dir_ent subsequently will get the updated attrs.

Reply:
alright, thanks